app/assets/javascripts/admin/variant_overrides/filters/import_date_filter.js.coffee

```diff
@@ -0,0 +1,12 @@
+angular.module("admin.variantOverrides").filter "importDate", ($filter, variantOverrides) ->
+  return (products, hub_id, date) ->
+    return [] if !hub_id
+    return $filter('filter')(products, (product) ->
+      return true if date == 0 or date == undefined or date == '0' or date == ''
+
+      for variant in product.variants
+        for vo in variantOverrides
```

Comment:
[forEach](https://docs.angularjs.org/api/ng/function/angular.forEach) feels more JSy, don't you think?